static/css/cf-theme-overrides.less

```diff
@@ -121,6 +121,9 @@
 // .error
 @input-error:                   @redorange;
 
+// .warning
+@input-warning:                 @gold;
```

Comment:
Ahh I see so that explains the two values. Perhaps add a TODO so that TODO changes can be searched for? What's the canonical source for `cf-theme-override.less` anyway? Is it what cf-generator has?